src/test/java/itm/oss/splitter/SettlementTest.java

```diff
@@ -0,0 +1,33 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SettlementTest {
+
+    @Test
+    void simpleTest() {
+        Balance b = new Balance();
+        b.put("Alice", new BigDecimal("-30"));
+        b.put("Bob", new BigDecimal("30"));
+
+        ArrayList<Payment> result = Settlement.suggest(b);
+
+        assertEquals(1, result.size());
+        assertEquals("Alice", result.get(0).getFrom());
+        assertEquals("Bob", result.get(0).getTo());
+        assertEquals(0, result.get(0).getAmount().compareTo(new BigDecimal("30")));
+    }
+
+    @Test
+    void tinyAmountTest() {
```

Comment:
The current name `tinyAmountTest` is a bit vague regarding the expected outcome of the test.

Following the naming convention of `should_DoSomething_WhenConditionIsMet` or `test_whenCondition_thenExpectedResult`, renaming the function would clarify the test's purpose without having to read all the function.

What do you think about one of these alternatives?

```suggestion
void test_ignoresPayments_whenBalanceIsBelowThreshold() {